service/build.gradle

```diff
@@ -15,7 +15,7 @@ apply from: 'publishing.gradle'
 dependencies {
     implementation "io.prometheus:simpleclient_httpserver:0.12.0"
     implementation "io.opencensus:opencensus-exporter-stats-prometheus:0.28.3"
-    implementation "org.broadinstitute.cromwell:cromwell-client_2.13:0.1-2eb5843a8-SNAP"
+    implementation "org.broadinstitute.cromwell:cromwell-client_2.13:0.1-1a484ba95-SNAP"
```

Comment:
I think the way to go about this is to submit a PR with those changes you made to `cromwell.yaml`, under the issue [WM-1491](https://broadworkbench.atlassian.net/browse/WM-1491). We can quickly get that merged in so that you have a commit on `develop` to use here.